datalad/support/annexrepo.py

```diff
@@ -955,8 +960,10 @@ def _run_annex_command(self, annex_cmd,
             If False, annex.merge-annex-branches=false config will be set for
             git-annex call.  Useful for operations which are not intended to
             benefit from updating information about remote git-annexes
+        runner: {None, "gitwitless"}, optional
+            Use specified runner class instead of the bound Runner instance.
         **kwargs
-            these are passed as additional kwargs to datalad.cmd.Runner.run()
+            these are passed as additional kwargs to the *Runner.run()
```

Comment:
It looks like the doc build is unhappy about this:

```
Warning, treated as error:
/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/datalad/cmd.py:docstring
of datalad.cmd.GitRunnerBase:1:Inline emphasis start-string without
end-string.
```

https://github.com/datalad/datalad/pull/4210/checks?check_run_id=474982172#step:6:144

Reply:
> > uff... might be -- would need to adjust then the helper to also deduce the target branch (could be a `maint`, not master)
> 
> If you configure your upstream branch, you could just always use `@{upstream}` (or `@{u}`).

oh, I am not yet that advanced I am afraid ;)  I guess will need to get into practice of doing that

Reply:
and thanks for the doc gotcha -- will push fix now